File: LeagueOfNews.Model/Website.cs
using System.Collections.Generic;

namespace LeagueOfNews.Model
{
    public class Website
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public IEnumerable<Website> Subpages { get; set; }
        public string Url { get; set; }
        public string ParserName { get; set; }
    }
}

File: league_of_news/lib/features/newsfeeds/domain/entities/website.dart
import 'package:league_of_news/features/newsfeeds/data/models/website_model.dart';
import 'package:meta/meta.dart';

class Website {
  final int id;
  final String name;
  final List<WebsiteModel> subpages;
  final String url;

  Website({
    @required this.id,
    @required this.name,
    @required this.subpages,
    @required this.url,
  });
}
